experimental/traffic-portal/src/app/core/statuses/statuses-table/statuses-table.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
This file is missing a license header. See other typescript files for a comment block at the top that you can copy.